addons/hr_holidays/models/hr_employee.py

```diff
@@ -268,6 +268,14 @@ def action_time_off_dashboard(self):
             },
         }
 
+    def _get_contextual_employee(self):
+        if self.env.context.get('employee_id'):
+            return self.browse(self.env.context.get('employee_id'))
```

Comment:
As you know it does exists in the dict:
```suggestion
            return self.browse(self.env.context['employee_id'])
```